src/modules/Api.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable lines-between-class-members */
```

Comment:
- [ ] It is not a good practice to disable linter rules because you are giving room for making `syntax errors` and this can introduce a lot of bugs in the codebase in the future. It is advisable to work toward correcting all the errors flagged by linters instead of disabling some of the rules. So please do not disable any linter rules in all your files rather correct them 😃 👍

Reply:
Ok. I am solving that also.

Reply:
I solved that. Now Api.js contains no  "eslint-disable" message.

The screenshot is attached.

https://user-images.githubusercontent.com/106825373/196935492-1f5e2e89-5d31-48db-8c0b-3c4e4d8fe397.png